code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -14,7 +14,21 @@
 	status_flags = GODMODE  // You can't damage it.
 	mouse_opacity = 0
 	see_in_dark = 7
-	invisibility = INVISIBILITY_AI_EYE
+	invisibility = SEE_INVISIBLE_OBSERVER

```

Comment:
Not sure if it matter to Paradise but on Bay I gave the AI eye a special invisibility level so that cultists wouldn't see it while using the seer/see invisibility rune (see_invis of 60 vs invis of 61).